overlord/ifacestate/export_test.go

```diff
@@ -49,6 +50,22 @@ func MockContentLinkRetryTimeout(d time.Duration) (restore func()) {
 	return func() { contentLinkRetryTimeout = old }
 }
 
+func MockCreateUDevMonitor(new func(udevmonitor.DeviceAddedFunc, udevmonitor.DeviceRemovedFunc) udevmonitor.Interface) (restore func()) {
+	old := createUdevMonitor
+	createUdevMonitor = new
+	return func() {
+		createUdevMonitor = old
+	}
+}
+
+func MockUdevInitRetryTimeout(t time.Duration) (restore func()) {
```

Comment:
UDev or Udev? The former seems nicer, but not both in either case.

The whole change needs reviewing after this. Above we already see more inconsistencies inside the function.